etc/behat/services/contexts.xml

```diff
@@ -143,6 +146,24 @@
             <tag name="sylius.behat.context" />
         </service>
 
+        <service id="sylius.behat.context.setup.taxonomy" class="%sylius.behat.context.setup.taxonomy.class%" scope="scenario">
+            <argument type="service" id="sylius.repository.taxon" container="symfony" />
+            <argument type="service" id="sylius.repository.taxonomy" container="symfony" />
+            <argument type="service" id="sylius.factory.taxonomy" container="symfony" />
+            <argument type="service" id="sylius.factory.taxon" container="symfony" />
+            <argument type="service" id="doctrine.orm.entity_manager" container="symfony" />
+            <tag name="sylius.behat.context" />
+        </service>
+
+        <service id="sylius.behat.context.transform.lexical" class="%sylius.behat.context.transform.lexical.class%" scope="scenario">
```

Comment:
I think github provides quite clear message, that not yet :smile: 
![zrzut ekranu 2016-02-29 o 16 09 05](https://cloud.githubusercontent.com/assets/6212718/13398494/d567b5c0-defe-11e5-9ce1-98798f749de5.png)